pkg/services/postgresql/catalog.go

```diff
@@ -103,6 +103,17 @@ func (m *module) GetCatalog() (service.Catalog, error) {
 					Bullets:     []string{"100 DTUs"},
 				},
 			}),
+			service.NewPlan(&service.PlanProperties{
```

Comment:
You also need to add this to the dbms only service. Not sure if you were waiting for validation on the comment below.

Reply:
Nope... this _is_ already under DBMS-only. It needs to be added under all-in-one. See [other comment](https://github.com/Azure/open-service-broker-azure/pull/390#issuecomment-380196937)

Reply:
yeah flipped that in my brain :)

Reply:
:+1: added to all-in-one in https://github.com/Azure/open-service-broker-azure/pull/390/commits/2bf20588332fe3bb60c0dd105742fca712032697